.github/workflows/c.yml

```diff
@@ -21,7 +21,16 @@ jobs:
     steps:
       - uses: actions/checkout@v4.1.1
       - name: Install libbpf
+        working-directory: bpfassets/libbpf
         run: |
-          sudo apt-get update && sudo apt-get install -y libelf-dev libbpf-dev
+          sudo apt remove libbpf-dev
+          sudo apt install libelf-dev
+          mkdir temp-libbpf
+          cd temp-libbpf
+          git clone https://github.com/libbpf/libbpf
+          cd libbpf/src
+          sudo make install_headers
+          sudo make install_uapi_headers
+          sudo prefix=/usr  BUILD_STATIC_ONLY=y make install
```

Comment:
shouldn't we checkout a particular version instead of main?

Reply:
we checkout main in the unit test code and I just copied that here.
this is only temporary until #1438 gets in when it will be removed.
I just wanted to get the build tags removed first to avoid bloating the cilium-ebpf PR more.